setup.py

```diff
@@ -21,7 +21,7 @@
     package_data={"src/pythonjsonlogger": ["py.typed"]},
     packages=find_packages("src", exclude="tests"),
     # https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires
-    python_requires='>=3.7',
+    python_requires='>=3.6.8',
```

Comment:
python3.6 is not maintained anymore: https://endoflife.date/python

The EOL was 1 year ago.